### PR TITLE
Add Prover Technology

### PR DESCRIPTION
**Pull request by danabr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ NASA | USA | Space | ```PVS``` [NASA Langley Formal Methods Research Program](ht
 [Nomadic Labs](https://nomadic-labs.com/) | Paris, France | blockchain | ```Coq``` [page on software verification](https://nomadic-labs.com/software_verification.html)
 [Oracle](https://www.oracle.com/) | Redwood Shores, CA, USA | Enterprise software, Cloud computing, Computer hardware | `ACL2` ([Proving Theorems about Java and the JVM with ACL2](https://www.cs.utexas.edu/users/moore/publications/marktoberdorf-02/index.html))
 [Particular Software](https://particular.net/) | | | ```TLA+``` [TLA+ Specifications for NServiceBus](https://github.com/tmasternak/NServiceBus.ModelChecking)
+[Prover Technology](https://www.prover.com/) | Europe | Railway | `Model checking`
```
Comment (ligurio) [inline README.md]: Put the line after Pingcap

Comment (danabr) [reply README.md]: Yes, of course. Fixed.

File: README.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ NASA | USA | Space | ```PVS``` [NASA Langley Formal Methods Research Program](ht
 [Oracle](https://www.oracle.com/) | Redwood Shores, CA, USA | Enterprise software, Cloud computing, Computer hardware | `ACL2` ([Proving Theorems about Java and the JVM with ACL2](https://www.cs.utexas.edu/users/moore/publications/marktoberdorf-02/index.html))
 [Particular Software](https://particular.net/) | | | ```TLA+``` [TLA+ Specifications for NServiceBus](https://github.com/tmasternak/NServiceBus.ModelChecking)
 PingCAP | | | ```TLA+``` [TLA+ in TiDB](https://github.com/pingcap/tla-plus)
+[Prover Technology](https://www.prover.com/) | Europe | Railway | `Model checking`
```
Comment (ligurio) [inline README.md:56]: Are there any public resources that prove using a model checking?
Papers, slides etc.

Comment (danabr) [reply README.md:56]: The blog has many articles about this (e.g. https://www.prover.com/verification-validation/psl-best-in-class-formal-verification-performance/ and https://www.prover.com/modeling/using-hll-to-solve-sudoku-puzzles/), but I can see if I can dig up some more tecnical academical papers.

**Comment by ligurio** [comment]
@danabr thanks!

**Pull request merged**